src/assets/languages/default-en.json

```diff
@@ -1,9 +1,11 @@
 {
+  "VENDOR" : "false",
   "SIDEBAR": {
     "HOME": "Overview",
     "DASHBOARD": "Ci/E2E Dashboard",
     "WORKLOADS": "Workload Dashboard",
-    "ORGANISATION": "MayaData.Inc"
+    "ORGANISATION": "MayaData.Inc",
+    "DASHBOARD-URL": "https://gitlab.com/geckovuser/"
```

Comment:
this is not need here

Reply:
This link is not used if vendor is false, we can pass this ant link or blank also